obspy/signal/tests/test_rotate.py

```diff
@@ -144,6 +145,26 @@ def test_rotate2zne_round_trip(self):
         self.assertTrue(np.allclose(n, n_new, rtol=1E-7, atol=1e-7))
         self.assertTrue(np.allclose(e, e_new, rtol=1E-7, atol=1e-7))
 
+    def test_rotate2zne_raise(self):
+        """
+        Check that rotate2zne() raises on unequal lengths of data.
+        """
+        z = np.ones(3, dtype=np.float64)
+        n = np.ones(5, dtype=np.float64)
+        e = np.ones(3, dtype=np.float64)
+
+        # Random values.
+        dip_1, dip_2, dip_3 = 0.0, 30.0, 60.0
+        azi_1, azi_2, azi_3 = 0.0, 170.0, 35.0
+
+        if PY2:
+            testmethod = self.assertRaisesRegexp
+        else:
+            testmethod = self.assertRaisesRegex
```

Comment:
Oh nice - I had no clue this existed!

If we end up using this more places we could generate an ObsPy `TestCase` class and fix it there.